pew/pew.py

```diff
@@ -188,7 +190,9 @@ def shell(env, cwd=None):
         if 'CMDER_ROOT' in os.environ:
             shell = 'Cmder'
         elif windows:
-            shell = 'powershell'
+            c = delegator.run('tasklist /fi "PID eq {0}" /fo csv /nh'.format(os.getppid()))
```

Comment:
This is the magic line.  
```python
c = delegator.run('tasklist /fi "PID eq {0}" /fo csv /nh'.format(os.getppid()))
```
That gets the parent id running the python executable.  it then submits it as a search argument to the windows util `tasklist`.  I'm using @kennethreitz's delegator module to run the commands for ease.